### PR TITLE
Add Jira Plugin Doc

### PR DESCRIPTION
**Pull request by teesloane**

### CODE REVIEW

File: jekyll/_cci2/jira-plugin.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- Project name
+- Issue type
+- Issue summary
+- Description
+
+![]({{ site.baseurl }}/assets/img/docs/jira_plugin_6.png)
+
+Note: The current JIRA plugin only supports default fields.
+
+You're all set to creating quick tickets from your build page!
```
Comment (michelle-luna) [inline jekyll/_cci2/jira-plugin.md]: is it the Jobs page (stated above) or the Build page?

File: jekyll/_cci2/jira-plugin.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+layout: classic-docs
+title: "Connecting JIRA with CircleCI"
+categories: [how-to]
+description: "Connecting JIRA with CircleCI"
+---
+
+This document describes how you can connect JIRA to your CircleCI builds. With
+the CircleCI JIRA plugin, you can create JIRA tickets directly from your Jobs
+page, allowing you assign tasks and fixes based on the status of your build.
```
Comment (michelle-luna) [inline jekyll/_cci2/jira-plugin.md]: allowing you assign -> allowing you to assign
should this be "status of your job" instead of "build"?

**Comment by michelle-luna** [comment]
thanks for doing this!

**Comment by teesloane** [comment]
Made the `build -> jobs` correction. It looks like we need to update the description of the jira plugin as it's referring to the `jobs` page as `builds`. Not sure who has access to make that change.

File: jekyll/_cci2/jira-plugin.md (hunk 1 above)
Comment (KunalJain) [reply jekyll/_cci2/jira-plugin.md]: Job output page 🙏

File: jekyll/_cci2/jira-plugin.md (hunk 2 above)
Comment (KunalJain) [reply jekyll/_cci2/jira-plugin.md]: Job please.

Comment (teesloane) [reply jekyll/_cci2/jira-plugin.md]: ok, will switch back!

File: jekyll/_cci2/jira-plugin.md (hunk 1 above)
Comment (teesloane) [reply jekyll/_cci2/jira-plugin.md]: fixed!

**Pull request merged**